### PR TITLE
Support whitespace

### PR DESCRIPTION
**Pull request by TrevorDArcyEvans**
Also a not in there to support other ASCII characters

### CODE REVIEW

**Review by mfe- — COMMENTED** [review]
Thanks for your PR. In which scenario didn't work the whitespaces for you? 
I tested the change with `Replace(" ","nbsp;")` which results in a lot of additional characters. I did a comparison of an app which uses the nuget with the version of your PR:
Generated html code
`version_check&nbsp;Checks&nbsp;the&nbsp;validity&nbsp;and&nbsp;type,&nbsp;of&nbsp;the&nbsp;specified&nbsp;version`
and with the old nuget version the generated html code looks like:
`version_check Checks the validity and type, of the specified version`

I think `Replace(" ","nbsp;")` should be configurable, so users could decide which output they want.

For example something like that `ReplaceWhitespace="true"`

```
    <Blzr.Console.BlazorConsole @ref="c"
                                Name="fooTest"
                                ShowRepositoryUrl="true"
                                AutoScroll="true"
                                SetAutoFocusToConsoleInput="true"
                                UseOriginalSystemConsole="false"
                                ReplaceWhitespace="true" />
```
 Looking forward to hear from you.

**Comment by TrevorDArcyEvans** [comment]
The issue with my change is that for *single* whitespaces, it would generate excessive output :-(  I guess this would only be an issue if someone wants to select the output and copy it to the clipboard.

AFAICT, the option of 'ReplaceWhitespace' would only really be useful for the case of single whitespaces and copying.  Otherwise, it's not clear to me if additional characters are undesirable.

TBH, I'm struggling to see why anyone would *not* want multiple whitespaces replaced.  Most console based apps use multiple spaces to align output eg _Lunar Lander_ above

File: README.md [hunk 1]
```diff
@@ -2,13 +2,8 @@
 
 The goal of this project is to make it easy to move existing console application to blazor web assembly.
 
-[![Console.Extensions package in get-the-solution feed in Azure Artifacts](https://feeds.dev.azure.com/get-the-solution/_apis/public/Packaging/Feeds/3bf81259-ccfe-4071-b8f8-bb5f44d4a8fb/Packages/89bd64e7-4569-44b5-946f-6830e6ac5694/Badge)](https://dev.azure.com/get-the-solution/get-the-solution/_packaging?_a=package&feed=3bf81259-ccfe-4071-b8f8-bb5f44d4a8fb&package=89bd64e7-4569-44b5-946f-6830e6ac5694&preferRelease=true)
```
[the remark this answers is not in the document]
Comment (TrevorDArcyEvans) [reply README.md:5]: Done!  (I think)

**Comment by mfe-** [comment]
Thanks for the explanation. 
> AFAICT, the option of 'ReplaceWhitespace' would only really be useful for the case of single whitespaces and copying.
I really would prefer having this configurable by a property for the above reason and because the output behavior will be changed. The property should by default use the old behavior (`ReplaceWhitespace=false`) and can be switched to your new implementation with `ReplaceWhitespace=true`

If you want, I can introduce this property and try to add it to your PR or you add it with an additional commit.

**Comment by TrevorDArcyEvans** [comment]
Please introduce the property and add it with an additional commit

**Comment by mfe-** [comment]
I'll introduce the property with a separate PR, since it looks it easier to do so

**Pull request merged**